newswires/client/src/WireItemList.tsx

```diff
@@ -26,11 +26,17 @@ export const WireItemList = ({
 	wires: WireData[];
 	totalCount: number;
 }) => {
-	const { config, loadMoreResults } = useSearch();
+	const { config, loadMoreResults, previousItemId } = useSearch();
 
 	const [isLoadingMore, setIsLoadingMore] = useState<boolean>(false);
 
-	const selectedWireId = config.itemId;
+	const [selectedWireId, setSelectedWireId] = useState<string | undefined>(
```

Comment:
why does `selectedWireId` need to be a piece of state here? shouldn't the component get re-rendered anyway when we change `config.itemId`?